fossunited/chapters/doctype/foss_event_rsvp_submission/test_foss_event_rsvp_submission.py

```diff
@@ -55,3 +55,39 @@ def test_add_to_email_group(self):
                 "Email Group Member", {"email": WEBSITE_USER, "email_group": email_group}
             )
         )
+
+    def test_acceptance_workflow(self):
+        # Given an RSVP form with accept all incoming responses
+        rsvp = self.rsvp
+
+        frappe.set_user(WEBSITE_USER)
+        # When a submission is done
+        submission = insert_rsvp_submission(linked_rsvp=rsvp.name)
+
+        # Then the submission status should be accepted
+        self.assertTrue(submission.status, "Accepted")
+
+    def test_pending_workflow(self):
+        # Given an rsvp with requires_host_approval = True
+        rsvp = self.rsvp
+        rsvp.requires_host_approval = True
+        rsvp.save()
+
+        frappe.set_user(WEBSITE_USER)
+        # When a submission is created
+        submission = insert_rsvp_submission(linked_rsvp=rsvp.name)
+
+        # Then the submission status should be pending
+        self.assertTrue(submission.status, "Pending")
```

Comment:
nitpicky comment - could we extend this test to confirm that the submission status can be updated to Accepted and it will be saved?